### ISSUE TITLE
How to install?

### ISSUE THREAD

**Issue by QuantumLiu** [xuduo35/StyleFlowPytorch#6]
Will you provide requirements.txt?

**Comment by xuduo35** [comment]
In my environment, python3 -m pip freeze. But not all the modules are needed, just for reference

absl-py==0.12.0
addict==2.4.0
aiohttp==3.8.1
aiosignal==1.2.0
albumentations==1.0.3
altair==4.1.0
antlr4-python3-runtime==4.8
argon2-cffi==21.1.0
arrow==1.2.1
astor==0.8.1
astunparse==1.6.3
async-timeout==4.0.2
attrs==21.2.0
audioread==2.1.9
backcall==0.2.0
backports.zoneinfo==0.2.1
base58==2.1.0
basicsr==1.3.4.9
beautifulsoup4==4.10.0
bleach==4.1.0
blinker==1.4
bokeh==2.4.2
braceexpand==0.1.7
Brotli==1.0.9
bson==0.5.10
bz2file==0.98
cachetools==4.2.2
certifi==2021.5.30
cffi==1.15.0
charset-normalizer==2.0.4
clang==5.0
click==7.1.2
clip @ git+https://github.com/openai/CLIP.git@40f5484c1c74edd83cb9cf687c6ab92b28d8b656
cloudpickle==2.0.0
configparser==5.1.0
cycler==0.10.0
dateparser==1.1.0
debugpy==1.5.1
decorator==5.1.0
defusedxml==0.7.1
diffdist==0.1
dill==0.3.4
dlib==19.22.1
dm-tree==0.1.6
docker-pycreds==0.4.0
dominate==2.6.0
dopamine-rl==3.0.1
easing-functions==1.0.4
efficientnet-pytorch==0.7.1
einops==0.4.1
entrypoints==0.3
face-alignment==1.3.5
facexlib==0.2.1.1
fairscale==0.4.5
faiss-gpu==1.7.2
filelock==3.6.0
filterpy==1.4.5
Flask==2.0.2
Flask-Compress==1.10.1
Flask-Cors==3.0.10
flatbuffers==1.12
frozenlist==1.3.0
fsspec==2022.2.0
ftfy==6.0.3
future==0.18.2
gast==0.4.0
gdown==4.4.0
gevent==21.12.0
gfpgan==0.2.4
gin-config==0.5.0
gitdb==4.0.9
GitPython==3.1.24
google-api-core==2.4.0
google-api-python-client==2.36.0
google-auth==1.35.0
google-auth-httplib2==0.1.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
googleapis-common-protos==1.54.0
greenlet==1.1.2
grpcio==1.39.0
gunicorn==20.1.0
gym==0.21.0
h5py==3.1.0
httplib2==0.20.2
huggingface-hub==0.4.0
idna==3.2
imageio==2.9.0
imageio-ffmpeg==0.4.5
imgaug==0.4.0
importlib-resources==5.4.0
intervaltree==3.1.0
ipykernel==6.4.2
ipython==7.28.0
ipython-genutils==0.2.0
ipywidgets==7.6.5
itsdangerous==2.0.1
jedi==0.18.0
Jinja2==3.0.2
joblib==1.1.0
jsonpatch==1.32
jsonpointer==2.2
jsonschema==4.1.2
jupyter==1.0.0
jupyter-client==7.0.6
jupyter-console==6.4.0
jupyter-core==4.8.1
jupyterlab-pygments==0.1.2
jupyterlab-widgets==1.0.2
kaggle==1.5.12
-e git+https://github.com/NVIDIAGameWorks/kaolin@a9b007a81debf2129110f3d27acbefa517262a54#egg=kaolin
keras==2.8.0
Keras-Preprocessing==1.1.2
kfac==0.2.0
kiwisolver==1.3.2
libclang==12.0.0
librosa==0.7.2
llvmlite==0.32.1
lmdb==1.2.1
lpips==0.1.4
Markdown==3.3.4
MarkupSafe==2.0.1
matplotlib==3.4.3
matplotlib-inline==0.1.3
mesh-tensorflow==0.1.19
mido==1.2.6
mistune==0.8.4
mmcv-full==1.3.16
mmsegmentation==0.18.0
moment==0.12.1
mpmath==1.2.1
multidict==6.0.2
munch==2.5.0
nbclient==0.5.4
nbconvert==6.2.0
nbformat==5.1.3
nest-asyncio==1.5.1
networkx==2.6.2
nltk==3.7
notebook==6.4.5
numba==0.49.1
numpy==1.22.2
oauth2client==4.1.3
oauthlib==3.1.1
omegaconf==2.1.2
onnx==1.11.0
onnx-simplifier==0.3.7
onnx-tf==1.10.0
onnxoptimizer==0.2.6
onnxruntime==1.11.0
onnxruntime-gpu==1.8.1
opencv-python==4.5.3.56
opencv-python-headless==4.5.3.56
opt-einsum==3.3.0
packaging==21.0
pandas==1.3.4
pandocfilters==1.5.0
parso==0.8.2
pathtools==0.1.2
pexpect==4.8.0
pickleshare==0.7.5
Pillow==8.3.1
prefetch-generator==1.0.1
pretrainedmodels==0.7.4
pretty-midi==0.2.9
prettytable==3.3.0
prometheus-client==0.11.0
promise==2.3
prompt-toolkit==3.0.21
protobuf==3.17.3
psutil==5.8.0
ptflops==0.6.6
ptyprocess==0.7.0
pyarrow==6.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser==2.20
pydeck==0.7.1
pyDeprecate==0.3.1
pydub==0.25.1
Pygments==2.10.0
pyparsing==2.4.7
pypng==0.0.21
pyrsistent==0.18.0
PySocks==1.7.1
python-dateutil==2.8.2
python-slugify==5.0.2
pytorch-lightning==1.5.10
pytz==2021.3
pytz-deprecation-shim==0.1.0.post0
PyWavelets==1.1.1
PyYAML==5.4.1
pyzmq==22.3.0
qtconsole==5.1.1
QtPy==1.11.2
regex==2021.11.10
requests==2.26.0
requests-oauthlib==1.3.0
resampy==0.2.2
rsa==4.7.2
sacremoses==0.0.47
scikit-image==0.18.3
scikit-learn==1.0.1
scipy==1.7.1
Send2Trash==1.8.0
sentry-sdk==1.4.3
Shapely==1.7.1
shortuuid==1.0.8
six==1.15.0
sk-video==1.1.10
sklearn==0.0
smmap==5.0.0
sortedcontainers==2.4.0
SoundFile==0.10.3.post1
soupsieve==2.3.1
sox==1.4.1
streamlit==1.1.0
subprocess32==3.5.4
sympy==1.9
tabulate==0.8.9
tensor2tensor==1.15.7
tensorboard==2.8.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.0
tensorflow==2.8.0
tensorflow-addons==0.15.0
tensorflow-datasets==4.4.0
tensorflow-gan==2.1.0
tensorflow-hub==0.12.0
tensorflow-io-gcs-filesystem==0.23.1
tensorflow-metadata==1.5.0
tensorflow-probability==0.7.0
termcolor==1.1.0
terminado==0.12.1
testpath==0.5.0
text-unidecode==1.3
tf-estimator-nightly==2.8.0.dev2021122109
tf-slim==1.1.0
thop==0.0.31.post2005241907
threadpoolctl==3.0.0
tifffile==2021.8.30
times==0.7
timm==0.4.12
tokenizers==0.11.6
toml==0.10.2
toolz==0.11.1
torch==1.9.0+cu111
torchaudio==0.9.0
torchcontrib==0.0.2
torchdiffeq==0.2.2
torchfile==0.1.0
torchmetrics==0.7.3
torchnet==0.0.4
torchvision==0.10.0+cu111
tornado==6.1
tqdm==4.62.2
traitlets==5.1.1
transformers==4.17.0
typeguard==2.13.3
typing_extensions==4.0.1
tzdata==2021.5
tzlocal==4.0.2
uritemplate==4.1.1
urllib3==1.26.6
usd-core==21.11
validators==0.18.2
visdom==0.1.8.9
wandb==0.12.6
watchdog==2.1.6
wcwidth==0.2.5
webdataset==0.1.103
webencodings==0.5.1
websocket-client==1.2.1
Werkzeug==2.0.1
widgetsnbextension==3.5.1
wrapt==1.12.1
yapf==0.32.0
yarl==1.7.2
yaspin==2.1.0
zipp==3.7.0
zope.event==4.5.0
zope.interface==5.4.0
